region/norfair/crocomire.json

```diff
@@ -791,13 +803,25 @@
             {
               "id": 4,
               "requires": [
-                {"canVisitNode": {
-                  "nodeid": 4,
-                  "scope": "room"
-                }}
+                {"or":[
+                  {"canVisitNode": {
+                    "nodeid": 4,
+                    "scope": "room"
+                  }},
+                  {"and":[
+                    "canSwim",
+                    {"canShineCharge": {
+                      "usedTiles": 31,
+                      "shinesparkFrames":35
+                    }},
+                    {"cost": [
+                      {"acidFrames": 130}
```

Comment:
Are you sure the acid version can be done using full 31 length charge? Looks like it would need a slower/shorter charge to not bonk something and lose the blue suit.

Reply:
I made sure to do it when getting my measurements

Reply:
Doing it with a bit less strict of a setup, it looks more like acid 140 frames though. Guess I can change that.